boards/arm/stm32wb5mm_dk/stm32wb5mm_dk.yaml

```diff
@@ -0,0 +1,14 @@
+identifier: stm32wb5mm_dk
+name: ST STM32WB5MM-DK Discovery Development Board
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 256
+flash: 1024
```

Comment:
This is the size available for application, so it should be 876K. Flashing a f/w larger than this will generate an error.
This field is only used by CI (in tree), and it is unlikely CI can generate this size of binary. So this is only for correctness.